README.md

```diff
@@ -175,3 +175,7 @@ sudo make install
 ## Troubleshooting
 
 See [doc/troubleshooting](doc/troubleshooting.md)
+
+## Community
+
+* Slack: #kube-spawn on https://kubernetes.slack.com/messages/C9ZMJH2NL/
```

Comment:
How about simply `Discuss the project on [Slack](https://kubernetes.slack.com/messages/C9ZMJH2NL/)`?

Reply:
That's a good idea. Fixed.